src/search/CMakeLists.txt

```diff
@@ -12,6 +12,14 @@ endif()
 ## == Project ==
 
 project(downward)
+
+# TODO: use multi-line strings to break up the long string when we switch to CMake 3.
+# https://stackoverflow.com/questions/7637539/how-to-split-strings-across-multiple-lines-in-cmake
+option(
+  USE_GLIBCXX_DEBUG
+  "When compiling with g++ enable the libstdc++ debug mode that does additional safety checks. The checks come at a performance cost and should only be enabled in debug mode. Additionally, enabling them will make the binary incompatible with libraries that are not compiled with this flag which can lead to hard-to-debug errors."
```

Comment:
Suggested new version: "Enable the libstdc++ debug mode that does additional safety checks. (On Linux systems, g++ and clang++ usually use libstdc++ for the C++ library.) The checks come at a significant performance cost and should only be enabled in debug mode. Enabling them makes the binary incompatible with libraries that are not compiled with this flag, which can lead to hard-to-debug errors."